store/src/db/tests.rs

```diff
@@ -18,12 +18,36 @@ fn create_db() -> Connection {
     conn
 }
 
+fn create_block(
+    conn: &mut Connection,
+    block_num: u32,
+) {
+    let block_header = BlockHeader::new(
+        num_to_rpo_digest(1),
+        block_num,
+        num_to_rpo_digest(3),
+        num_to_rpo_digest(4),
+        num_to_rpo_digest(5),
+        num_to_rpo_digest(6),
+        num_to_rpo_digest(7),
+        num_to_rpo_digest(8),
+        9_u8.into(),
```

Comment:
Why are we using 9_u8.into() here instead of num_to_rpo_digest ?

Reply:
I _think_ we don't have a `Into` conversion from u8 to RpoDigest. I think this is just wrapping the type to create the block header

Reply:
What do you call `wrapping` ?

Reply:
when you have a unit type struct, wrapping an inner type.